Makefile

```diff
@@ -142,3 +143,19 @@ clean:
 	rm -rf .cache
 	rm -rf $(BIN)
 	rm -rf $(RELEASE)
+
+# Package for Debian
+debpackage: DEBVERSION ?= $(shell echo $(VERSION) | sed "s/^v//" | \
+			sed -E "s/(.*)-(g[a-fA-F0-9]{6,8})(.*)/\1\3~\2/" | \
+			sed "s/-/~/")-$(shell grep -o "^VERSION_CODENAME=.*" /etc/os-release | \
+			cut -d= -f2-)
+debpackage:
+	rm -f debian/changelog
+	dch --create --package neofs-node -M -v $(DEBVERSION) \
+		-D $(shell sed -n "s/^VERSION_CODENAME=//p" /etc/os-release) \
```

Comment:
This doesn't work for `sid`, btw.

Reply:
sorry sid is not the target I test for

Reply:
Let's assume current Debian stable as a target platform.

Reply:
I expect to be able to build package for Debian V+, that's kinda the norm and regular packages are built for/pushed into the `sid` initially. But maybe a bit less dynamic `debian/changelog` can solve this.

Reply:
We have certain requirements for package version, and while the codename is not required per se it is a good option to make it clear. I'd rather keep it that way instead of re-evaluate versioning.